x-pack/plugins/security_solution/public/detections/components/rules/step_define_rule/schema.tsx

```diff
@@ -626,7 +626,7 @@ export const schema: FormSchema<DefineStepRule> = {
     label: i18n.translate(
       'xpack.securitySolution.detectionEngine.createRule.stepDefineRule.suppressionMissingFieldsLabel',
       {
-        defaultMessage: 'If “Suppress by” field does not exist',
+        defaultMessage: 'If a suppression field missing',
```

Comment:
nit: Update to match `ALERT_SUPPRESSION_MISSING_FIELDS_FORM_ROW_LABEL`, though TBH, I wasn't able to get this string to show up in the UI... 🤷‍♂️ 

```suggestion
        defaultMessage: 'If a suppression is field missing',
```

Reply:
`If a suppression field is missing` ?

Reply:
Just like `ALERT_SUPPRESSION_MISSING_FIELDS_FORM_ROW_LABEL` in the translations file